gb.yaml

```
@@ -67,6 +67,25 @@ months:
     regions: [gb]
     week: -1
     wday: 1
+    year_ranges:
+      until: 2021
+  - name: Bank Holiday
+    regions: [gb]
+    week: -1
+    wday: 1
+    year_ranges:
+      from: 2023      
+  6:
+    name: Bank Holiday
```

Is this right? The subelements of "6" are different - one is an array, one a hash entry.

Should it be:

```suggestion
  - name: Bank Holiday
```

---

@hlascelles ah, thanks! I messed up the yaml syntax here.